Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -513,11 +513,15 @@ void SVertexer::buildT2V(const o2::globaltracking::RecoContainer& recoData) // a
 
       // get Nclusters in the ITS if available
       int8_t nITSclu = -1;
+      bool shortOBITSOnlyTrack = false;
       auto itsGID = recoData.getITSContributorGID(tvid);
       if (itsGID.getSource() == GIndex::ITS) {
         if (isITSloaded) {
           auto& itsTrack = recoData.getITSTrack(itsGID);
           nITSclu = itsTrack.getNumberOfClusters();
+          if( itsTrack.hasHitOnLayer(6) && itsTrack.hasHitOnLayer(5) && itsTrack.hasHitOnLayer(4) && itsTrack.hasHitOnLayer(3)) {
```

Comment:
@shahor02 so Felix used the snippet `itsABTracklet.getFirstEntry() > 3;` for this, which is admittedly much better than my ugly check, but should it not actually have been `itsABTracklet.getFirstEntry() >= 3;` if the first layer is layer zero?...

Reply:
@ddobrigk , as I wrote in the comment to the previous PR, `itsABTracklet.getFirstEntry()` is wrong since has nothing to do with the ITS layers: this is the entry of the tracklet's 1st cluster index in the container of indices.
In opposite, `itsTrack.getFirstClusterLayer()` does provide the 1st contributing layer, and for the OB tracks one should indeed use `itsTrack.getFirstClusterLayer() >= 3`.

Reply:
Ah - sorry, I missed that comment. So better indeed check with `getFirstClusterLayer`: I've changed that! Thanks!